lib/raidenclient/RaidenClient.ts

```diff
@@ -135,11 +135,16 @@ class RaidenClient extends BaseClient {
   }
 
   public getRoutes =  async (_amount: number, _destination: string) => {
-    return []; // stub placeholder, query routes not currently implemented in raiden
+    // stub placeholder, query routes not currently implemented in raiden
+    return Promise.resolve([
```

Comment:
Since it's `async` you don't need to wrap the array in `Promise.resolve()`, and we're just returning plain synchronous values from other `async` methods in the codebase. It doesn't make any real difference but might as well make it consistent.

Reply:
Changed. Old habits die hard :-).